test/suites/dev-frontier-template/test-pov/test-evm-over-pov.ts

```diff
@@ -99,8 +99,8 @@ describeSuite({
                 // The block still contain the failed (out of gas) transaction so the PoV is still included
                 // in the block.
                 // 1M Gas allows ~250k of PoV, so we verify we are within range.
```

Comment:
This line is specially suspicious, 1M gas should indeed account for 250K pov proof, and our tests clearly indicate they dont